ci/ubuntu-22.04/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM ubuntu:22.04
+
+ENV DEBIAN_FRONTEND="noninteractive" TZ="America/Los_Angeles"
+
+# A version field to invalide Cirrus's build cache when needed, as suggested in
+# https://github.com/cirruslabs/cirrus-ci-docs/issues/544#issuecomment-566066822
+ENV DOCKERFILE_VERSION 20220614
+
+RUN apt-get update && apt-get -y install \
+    bc \
+    bison \
+    bsdmainutils \
+    ccache \
+    cmake \
+    curl \
+    flex \
+    g++ \
+    gcc \
+    git \
+    lcov \
+    libkrb5-dev \
+    libmaxminddb-dev \
+    libpcap-dev \
+    libssl-dev \
+    make \
+    python3 \
+    python3-dev \
+    python3-pip\
+    ruby \
+    sqlite3 \
+    swig \
+    unzip \
+    wget \
+    zlib1g-dev \
+  && apt autoclean \
+  && rm -rf /var/lib/apt/lists/*
+
+RUN pip3 install websockets junit2html
+RUN gem install coveralls-lcov
```

Comment:
The lcov gem only needs to be installed on whatever ubuntu version we're using for the sanitizer build, since that's the only place the coverage stuff runs. That said, I guess it's fine to install it on all of the ubuntu ones just in case we change which VM that uses.